.travis.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         - go mod vendor
       script:
         # Run multinamespace tests as admin (skip role settings check)
-        - make test PARALLEL_COUNT=2 && kubectl config use-context $TESTING_CONTEXT && make multinamespace-test
+        - make test PARALLEL_COUNT=2 && make batch-test && kubectl config use-context $TESTING_CONTEXT && make multinamespace-test
```

Comment:
@ryanemerson would you like to split `batch-test` with the separate Stage?

Reply:
With you saying this requires an additional VM I thought this would be overkill, in the same way that we don't create a separate stage for the multinamespace-tests.

Reply:
Sorry, I was talking about separating multinamespace-test, but not really know how long take Batch CR test. Probably we not needed a separate Stage for it. It depends of test time and case separating.

Reply:
I think six tests are enough to have a separate parallel CI stage. wdyt?